src/serialization.cpp

```diff
@@ -215,6 +217,8 @@ static void serializeContent(::inference::ModelInferResponse::InferOutputTensor&
         SERIALIZE_BY_DATATYPE(mutable_uint_contents, uint8_t)
     } else if (responseOutput.datatype() == "FP64") {
         SERIALIZE_BY_DATATYPE(mutable_fp64_contents, double)
+    } else if (responseOutput.datatype() == "STRING") {
+        responseOutput.mutable_contents()->add_bytes_contents((char*)tensor.data(), tensor.get_byte_size());
```

Comment:
@bstrzele  Why are you performing string serialization in this PR? It is out of scope in this release. What does tensor.data() return when the type is string?